run_forwarder.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+export IIP_UID=$(id -u iip)
+export IIP_GID=$(id -g iip)
+export KRB5CCNAME_FILE=/tmp/krb5cc_$(id -u iip)
+
+error_msg="$0: missing argument: -d [latiss|comcam] -s [summit|nts]"
+
+export IIP_CONFIG_DIR=
+export IIP_DEVICE=
+export IIP_SITE=
+export IIP_CONTAINER=
+
+while getopts d:s: option
+do
+case "${option}"
+in
+d) 
+    if [ "$OPTARG" = "latiss" ] || [ "$OPTARG" = "comcam" ]; then
+        IIP_DEVICE=${OPTARG}
+    else
+        echo "-d argument must be latiss or comcam"
+        exit 1
+    fi;;
+s) 
+    if [ "$OPTARG" = "summit" ] || [ "$OPTARG" = "nts" ]; then
+        IIP_SITE=${OPTARG}
+    else
+        echo "-s argument must be summit or nts"
+        exit 1
+    fi;;
+esac
+done
+
+if [[ -z $IIP_DEVICE ]] || [[ -z $IIP_SITE ]]; then
```

Comment:
Though it doesn't look like either `IIP_DEVICE` or `IIP_SITE` contain "weird" characters, I would double quote them just in case and to keep things consistent (i.e. other variable is double quoted in if-statements are double quoted).